### PR TITLE
Fix Scala3 SDK

### PR DESCRIPTION
**Pull request by jaakkonarhi**
# Description of the PR

+

# Testing
<table>
  <tr>
    <th>unit</th>
    <th>integration</th>
    <th>manual</th>
  </tr>
  <tr>
    <td>
      <ul>
        <li>- [ ] new <b>unit</b> tests created</li>
        <li>- [ ] all <b>unit</b> tests pass</li>
      </ul>
    </td>
    <td>
      <ul>
        <li>- [ ] new <b>integration</b> tests created</li>
        <li>- [ ] all <b>integration</b> tests pass</li>
      </ul>
    </td>
    <td>
      <ul>
        <li>- [ ] <b>manual</b> testing went well</li>
      </ul>
    </td>
  </tr>
</table>  
  
# Have you updated the [TESTING.md](https://github.com/Aalto-LeTech/aplus-courses/blob/master/TESTING.md) or other relevant documentation on _your_ branch?

- [ ] Yes.
- [ ] Not yet. I will do it next.
- [x] Not relevant.

### CODE REVIEW

File: src/main/java/fi/aalto/cs/apluscourses/intellij/model/Scala3Sdk.java [hunk 1]
```diff
@@ -29,6 +29,6 @@ public Scala3Sdk(@NotNull String name,
   @Override
   @NotNull
   protected String @NotNull [] getClassRoots() {
-    return new String[] {"scala3-library_" + scalaVersion + "-" + scalaVersion + ".jar"};
+    return new String[] {"scala3-library_3-" + scalaVersion + ".jar"};
```
Comment (OlliKiljunen) [inline src/main/java/fi/aalto/cs/apluscourses/intellij/model/Scala3Sdk.java:32]: So those JARs are like `scala3-library_3-3.0.0` ?

**Review by OlliKiljunen — APPROVED** [review]
Good! 👍

File: src/main/java/fi/aalto/cs/apluscourses/intellij/model/Scala3Sdk.java (hunk 1 above)
Comment (jaakkonarhi) [reply src/main/java/fi/aalto/cs/apluscourses/intellij/model/Scala3Sdk.java:32]: yep

File: src/main/java/fi/aalto/cs/apluscourses/intellij/model/CommonLibraryProvider.java [hunk 2]
```diff
@@ -35,10 +35,10 @@ public void setInitializationCallback(
 
   @Nullable
   protected Library createLibrary(@NotNull String name) {
-    if (name.startsWith("scala-sdk-")) {
+    if (name.startsWith("scala-sdk-3")) {
+      return new Scala3Sdk(name, name.replace("scala-sdk-", ""), project);
```
[the remark this answers is not in the document]
Comment (jaakkonarhi) [reply src/main/java/fi/aalto/cs/apluscourses/intellij/model/CommonLibraryProvider.java:39]: I think this is good enough

**Review by stellatsv — APPROVED** [review]

**Pull request merged**